superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FiltersConfigForm.tsx

```diff
@@ -599,9 +601,11 @@ const FiltersConfigForm = (
                       if (hasValue) {
                         return Promise.resolve();
                       }
-                      return Promise.reject(
-                        new Error(t('Default value is required')),
-                      );
+                      return Promise.resolve();
+                      // TODO: Some values does default value empty and disabled so there is no value
+                      // return Promise.reject(
+                      //   new Error(t('Default value is required')),
+                      // );
```

Comment:
I don't think we need to make this change - AFAIK it's enough to set the "default to first value" option and save, so no need to check the "default value" checkbox. I'd rather state in the description tooltip of the "default to first value" option that when using this option, setting a default value is not possible.

Reply:
current logic of defaultToFirstValue, requires to leave default value as `undefined` so it will fail on this validation, we can put some specific value here because it can different for different users